R/paren_body_linter.R

```diff
@@ -0,0 +1,33 @@
+#' @describeIn linters check that there is a space between the
+#' right parenthesis and the function body when no braces are used
+#' to define a function.
+#'
+#' @export
+paren_body_linter <- function() {
+  Linter(function(source_file) {
+    if (is.null(source_file$xml_parsed_content)) return(NULL)
+    
```

Comment:
Can you configure your IDE to remove trailing whitespace?
There are a lot of lints due to superfluous whitespace.

Reply:
I ran lintr on the files, they are clear for me now.